docs/mmteb/readme.md

```diff
@@ -0,0 +1,64 @@
+# Welcome to MMTEB! 👋
+
+MMTEB is a community-led extension of [MTEB](https://arxiv.org/abs/2210.07316) to cover embedding tasks for a massive number of languages.
```

Comment:
Maybe add explicitly the meaning of the MMTEB acronym ? I can't tell myself if it's "Massive Multi-lingual Text Embedding Benchmark" or "Multi-lingual Massive Text Embedding Benchmark" 😅

Reply:
Thanks! We are discussing the naming over at:
https://github.com/embeddings-benchmark/mteb/discussions/277

Would love your feedback on it.